08-interop/README.md

```diff
@@ -281,3 +281,188 @@ guide on Zephyr's documentation.
    dst_l2addr: 32:A9:FA:65:10:6B:11:14
    ~~ PKT    -  4 snips, total size:  85 byte
    ```
+   
+Task #12 - GNRC Border Router with WiFi uplink to the Internet
+========================
+### Description
+
+This test will ensure RIOT can connect to a real IPv6 enabled WiFi network and
+share the uplink connection with constrained nodes.
+
+An esp* board with the `gnrc_border_router` connects to your WiFi.
+The module `sock_dns` is used to resolve domain names.
+
+Two network interfaces are configured:
+
+ - a 802.11 WiFi interface used as uplink
+ - a proprietary `esp_now` interface as 6LoWPAN downlink
+ 
+ Materials
+---------
+
+You'll need:
+
+ - a pair of esp8266/esp32 boards
+ - 2.4 GHz WiFi network with an IPv6 uplink
+ - to make sure prefix delegation (IA_PD) is enabled in your router's DHCPv6 server
+   (On the popular Fritz!Box line of routers this can be enabled in the
+   Network -> Network Settings -> IPv6 Addresses menu)
+
+### Details
+
+Flash the `gnrc_border_router` example onto one of the esp* boards.
+Use the `sock_dns` module to enable name resolution.
+The credentials for the WiFi network will be passed on the command line.
+Replace `esp8266-esp-12x` with the esp* board of your choice, adjust `PORT` if needed.
+
+```
+USEMODULE=sock_dns make -C examples/gnrc_border_router BOARD=esp8266-esp-12x UPLINK=wifi WIFI_SSID='your_ssd' WIFI_PASS='your_password' PORT=/dev/ttyUSB0 flash term
```

Comment:
```suggestion
USEMODULE=sock_dns make -C examples/gnrc_border_router BOARD=esp<...> UPLINK=wifi WIFI_SSID=<your_ssd> WIFI_PASS=<your_password> PORT=<port> flash term
```

This way it should be clear what is variable or not and you can adjust the last sentence above accordingly.